packages/utilities/src/guards/Rate Limiter/RateLimit.ts

```diff
@@ -15,15 +15,15 @@ import { TimeOutEntry } from "./logic/index.js";
  * @param timeout - the time unit to use
  * @param value - the value for the time unit
  * @param message - the message to post when a command is called when the
- * user is in rate limit, defaults = "message being rate limited!"
+ * user is in rate limit, defaults = "message being rate limited!, please try again at {until}". use the placeholder {until} in your string to get time you can next call it `<t:epoch:T>`
```

Comment:
```suggestion
 * user is in rate limit, defaults = "message being rate limited!, please try again at {until}".
 * use the placeholder {until} in your string to get time you can next call it `<t:epoch:T>`
```

Reply:
should be `use the placeholder {until} in your string to get the time you can next call it `<t:epoch:T>``